host/src/FunctionsNetHost/AppLoader/NetHost.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Runtime.InteropServices;
+
+namespace FunctionsNetHost
+{
+    internal class NetHost
+    {
+        [DllImport("nethost", CharSet = CharSet.Auto)]
+        private static extern int get_hostfxr_path(
```

Comment:
https://github.com/dotnet/runtime/blob/main/docs/design/features/native-hosting.md#new-host-binary-for-finding-hostfxr

Found that this method always return the latest (If a preview version exists, that will be returned). In one machine I had net6,7 and 8 preview and I got 8 preview path.